magento/adobe-stock-integration
# Remove obsolete content after content synchronization

### Description (*)


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1474: The "Used in" still displays that the image is used in a Product after the Product has been deleted
2. ...

### Manual testing scenarios (*)
1. Create a Product and add an image from Media Gallery
2. Go to Content - Media Gallery and View the Details of the image that was previously added to the Product
3. Verify that the Details display that the image is **Used In 1 Product**
4. Go to _Catalog - Products_ and **Delete** the previously created Product
5. Perform ``` bin/magento indexer:reindex, bin/magento media-content:sync, bin/magento cache:clean``` 
6. Go to Go to Content - Media Gallery and View the Details of the image that was previously Used

The image is not Used in a product so it should not be displayed in Details
